sdk/appconfiguration/app-configuration/review/app-configuration.api.md

```diff
@@ -20,7 +19,7 @@ export interface AddConfigurationSettingOptions extends OperationOptions {
 export type AddConfigurationSettingParam<T extends string | FeatureFlagValue | SecretReferenceValue = string> = ConfigurationSettingParam<T>;
 
 // @public
-export interface AddConfigurationSettingResponse extends ConfigurationSetting, SyncTokenHeaderField, HttpResponseField<SyncTokenHeaderField> {
+export interface AddConfigurationSettingResponse extends ConfigurationSetting, SyncTokenHeaderField {
```

Comment:
Did we decide it isn't breaking to remove this _response field?

Reply:
I think so, but as @xirzec said,

> Yeah, since we documented this in the exported types, we either need a major bump (in which case we can ditch the compat dependency) or we have to wait for Sarangan's work to support _response in the shim.

"since we documented"